asset_dashboard/static/js/components/PortfolioTable.js

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 import ReactTable from './BaseTable'
+import columns from './table-utils/projectColumns'
```

Comment:
Import these default column values so we can reuse in both `ProjectsTable` and this `PortfolioTable`. We could put those values in the parent `PortfolioPlanner` container but I thought that would add too much noise to that component, especially since it's already doing so much.